lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -152,6 +153,133 @@ class ConvertAtenFlipOp : public OpConversionPattern<AtenFlipOp> {
 };
 } // namespace
 
+// Returns updated output dimension and scale_factor converted to integer.
+static std::pair<Value, Value> getScaledDims(OpBuilder &builder, Location loc,
```

Comment:
Getting the `scale_factor` converted to an int seems to be unrelated to the goal of this function, which is to scale the dims. I think it would be clearer to have this function simply return the scaled dim, and have the conversion to `int` of `scale_factor` happen in the lowering itself (since it's just one line of code)